frontend/test/metabase/visualizations/components/settings/ChartSettingFieldPicker.unit.spec.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import { within } from "@testing-library/dom";
+import { renderWithProviders } from "__support__/ui";
+
+// these tests use ChartSettings directly, but logic we're testing lives in ChartNestedSettingSeries
```

Comment:
Not sure what you mean by we're testing the logic in `ChartNestedSettingSeries`?

This PR changes` ChartSettings.jsx` code so that it doesn't crash when a column's semantic is currency.

In the [Slack thread](https://metaboat.slack.com/archives/C505ZNNH4/p1665002329746859), you mentioned this has something to do with `hasColumnSettings`, which is also in `ChartSettings.jsx`.

My question is that shouldn't this test be in `ChartSettings.unit.spec.js` instead?

Reply:
Ah geez. That should say `ChartSettingsFieldPicker`, not `ChartNestedSeriesSettings`.

The function that was causing the issue is defined in ChartSettings, but is passed down to ChartSettingFieldPicker (and currently it's the only widget that uses the prop). It's all kinda coupled :/

Reply:
That makes sense now. 👍